scapy/sendrecv.py

```diff
@@ -337,14 +340,51 @@ def sendpfast(x, pps=None, mbps=None, realtime=None, loop=0, file_cache=False, i
 
     if loop:
         argv.append("--loop=%i" % loop)
-        if file_cache:
-            argv.append("--preload-pcap")
+    if file_cache:
```

Comment:
I think this was a bug that existed before. `file_cache` should not be dependent on `loop`